x-pack/filebeat/input/awss3/config.go

```diff
@@ -33,6 +34,9 @@ type config struct {
 	AWSConfig           awscommon.ConfigAWS  `config:",inline"`
 	FileSelectors       []fileSelectorConfig `config:"file_selectors"`
 	ReaderConfig        readerConfig         `config:",inline"` // Reader options to apply when no file_selectors are used.
+	PathStyle           bool                 `config:"path_style"`
+	RegionOverride      string               `config:"region_override"`
```

Comment:
I would assume S3 compatible buckets support https://docs.aws.amazon.com/AmazonS3/latest/API/API_GetBucketLocation.html

Could you help me validate this?
`aws s3api --endpoint=%ENDPOINT% get-bucket-location --bucket %BUCKET_NAME%`
Does it return the following?
```
{
    "LocationConstraint": "%REGION%"
}
```

If so we should not need `RegionOverride` but just rely on `getRegionForBucketARN` also for not AWS S3 buckets

Reply:
Possibly.  I know minio has a region setting. I'll change it and see what happens.

Reply:
yes minio does and looks like all the s3 services i found also do.